Fls.Supervision.Api/Services/GithubMetricsService.cs

```diff
@@ -0,0 +1,26 @@
+using Newtonsoft.Json;
+using Convey.CQRS.Events;
+using System;
+using Fls.Supervision.Api.Commands;
+namespace Fls.Supervision.Api.Services 
+{
+    class GithubMetricsService
+    {
+        static IEvent ParseGithubEvent(string data) 
+        {
```

Comment:
It won't be a `string` passed here. Rather a well-prepared data class like `ReviewCreatedInfo : IGithubEventInfo`. It's not just a JSON sent from GitHub. The JSON will be parsed automatically by the endpoint to fill a class from #3 and then the handler will do its best to turn `ProcessGithubEvent` command int some implementation of the `IGithubEventInfo`. And then the service will use the event info to obtain some data useful in computing metrics from the event and return these data.